### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by bluluke**
GET /api/topics return array of objects.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  test('200: responds with array of all topic objects with both slug and description properties', () => { 
+    
+    return request(app)
+    .get("/api/topics")
+    
+    .expect(200)
+    .then((topics) => { 
+      const { body } = topics;
+      
+      expect(body).toHaveLength(3);
```
Comment (johnhutcheon) [inline __tests__/app.test.js:26]: Nice work checking the length before a forEach 👍

File: db/app.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
```
Comment (johnhutcheon) [inline db/app.js:3]: Is this line of code doing anything at the moment? If not, please remove until it's useful 🙂

File: db/controllers/topicsController.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+
+
+const { selectTopics } = require('../models/topicsModel')
+
+exports.getTopics = (req, res, next) => {
+    console.log(req, 'req in controller');
```
Comment (johnhutcheon) [inline db/controllers/topicsController.js:6]: Please remove any console logs before submitting PRs. 👍

File: db/models/topicsModel.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+
+const db = require('../connection');
+
+exports.selectTopics = (arg) => {
```
Comment (johnhutcheon) [inline db/models/topicsModel.js:4]: Could we avoid including 'arg' in this instance as it's not doing anything within the function?

**Review by johnhutcheon — APPROVED** [review]
Hi Luke, nice work! Happy for you to merge, but please see my comments for future reference! 😄

File: db/app.js (hunk 2 above)
Comment (bluluke) [reply db/app.js:3]: Thanks for the feedback c: I thought the first 2 lines were needed at least for the routes to work?

Comment (bluluke) [reply db/app.js:3]: I have understood now. Thanks!

**Pull request merged**